core/web_admin.py

```
@@ -0,0 +1,1040 @@
+            "telegram_session",
+        }
+
+        for key in root_fields:
+            if key not in incoming:
+                continue
+            value = incoming[key]
+            if key == "api_id":
+                try:
+                    value = int(value or 0)
+                except (TypeError, ValueError):
+                    raise WebAdminError("api_id 必须是数字。")
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)